justifi-tech/web-component-examples
# Migrate from NPM to PNPM

This PR migrates from NPM to PNPM 

#10 

**Steps for QA**
[ ] - Run `pnpm install` 
[ ] - Please check if `.env` file has valid values before running the app.
[ ] - Run `pnpm run start:<options>`. It should run the app successfully.
